### DISCUSSION TITLE
Support installation without docker

### DISCUSSION THREAD

**Discussion by leona-ya** [thomseddon/traefik-forward-auth#184] [Ideas]
Currently the only installation method specified in the README is docker. For me as not docker enthusiast it would be much better to support an alternative installation method. 

The way I went was to build a binary but the steps for it could I only find by reading the  Dockerfile.

I would propose to do at least one of the following two things:

* Create binary files for each release (as traefik does; [example release](https://github.com/traefik/traefik/releases/tag/v2.2.11))
* Add the steps for self compiling in the README

**Comment by thomseddon** [comment]
Interesting, it's currently designed to work with traefik, which I thought was only really used in containers - could I ask what you're use case is without docker?

**Comment by leona-ya** [comment]
I use traefik as reverse proxy with more features in special the middlewares, the easy way to use LE certificates and the easy way to configure it. 

I don't use docker since I run traefik in a [LXC Contianer](https://pve.proxmox.com/wiki/Linux_Container) on my Proxmox which don't have a good Docker support. And also I think for my use case Docker would be an overkill.

**Comment by thomseddon** [comment]
I see - this certainly seems to be quite a niche use-case, if there's a way I can automate this with github actions then I'd be up for it?

**Comment by leona-ya** [comment]
I created the pull reqeust #199 in which a GitHub Actions workflow is added. Maybe I will also have the motivation in the next days to write some examples for it or more text in the README.

I'm not an expert for GitHub Actions, so if you want me to change something, I can do it..

Here is an example release I made: https://github.com/em0lar/traefik-forward-auth/releases/tag/v2.2.0-test
